packages/documentation-framework/layouts/sideNavLayout/sideNavLayout.js

```diff
@@ -27,6 +27,8 @@ import {
 } from '@patternfly/react-core';
 import BarsIcon from '@patternfly/react-icons/dist/esm/icons/bars-icon';
 import GithubIcon from '@patternfly/react-icons/dist/esm/icons/github-icon';
+import AlignLeftIcon from '@patternfly/react-icons/dist/esm/icons/align-left-icon';
+import AlignRightIcon from '@patternfly/react-icons/dist/esm/icons/align-right-icon';
```

Comment:
I think you can remove these now.

Reply:
Thanks - pulled out 3 other unused image imports along with these ✅